modules/swagger-parser/src/main/java/io/swagger/parser/SwaggerResolver.java

```diff
@@ -53,4 +63,26 @@ public Swagger resolve() {
 
         return swagger;
     }
+
+    public static class Settings {
```

Comment:
> So the parser is essentially copying the parameters from the path-item object to the operations "below" it in hierarchy correct

Exactly

> you don't want it to copy them, or you want to copy and leave them there?

I don't want to copy them at all.

Before, we were using the swagger-core Object-Mapper, but it didn't support multi-files. So we compare it with swagger-parser and I tried to fix all important difference with this PR.